### PR TITLE
Create models.py

### PR DESCRIPTION
**Pull request by relssek98**
Described a new feature

### CODE REVIEW

**Comment by pthomas76** [comment]
You can remove the select import & the relationship import.

**Comment by pthomas76** [comment]
ForeignKey can be removed too.

**Comment by pthomas76** [comment]
Looks good to me!

File: database/models.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
+from typing import List
```
Comment (DXGatech) [inline database/models.py:1]: Suggestion (non-blocking): since these two imports are not used, we can remove these when refactoring code.

**Pull request merged**